proto/pocket/shared/service.proto

```diff
@@ -8,22 +8,18 @@ option go_package = "github.com/pokt-network/poktroll/x/shared/types";
 
 // TODO_CLEANUP(@Olshansk): Add native optional identifiers once its supported; https://github.com/ignite/cli/issues/3698
 
-// ServiceId message to encapsulate unique and semantic identifiers for a service on the network
-message ServiceId {
-    // NOTE: `ServiceId.Id` may seem redundant but was desigtned created to enable more complex service identification
+// Service message to encapsulate unique and semantic identifiers for a service on the network
+message Service {
     // For example, what if we want to request a session for a certain service but with some additional configs that identify it?
     string id = 1; // Unique identifier for the service
 
     // TODO_TECHDEBT: Name is currently unused but acts as a reminder than an optional onchain representation of the service is necessary
     string name = 2; // (Optional) Semantic human readable name for the service
-
-    // NOTE: `ServiceId.Id` may seem redundant but was designed to enable more complex service identification.
-    // For example, what if we want to request a session for a certain service but with some additional configs that identify it?
 }
 
 // ApplicationServiceConfig holds the service configuration the application stakes for
 message ApplicationServiceConfig {
-    ServiceId service_id = 1; // Unique and semantic identifier for the service
+    Service service = 1; // The Service for which the application is configured for
```

Comment:
NIT: I think it reads slightly better if we  drop  the "for" at the end:

```suggestion
    Service service = 1; // The Service for which the application is configured
```

Reply:
Didn't notice this so will tend to it in #123